README.md

```diff
@@ -119,7 +119,7 @@ USAGE
 * [`dvc features`](docs/features.md) - Create, view, or modify Features with the Management API.
 * [`dvc generate`](docs/generate.md) - Generate Devcycle related files.
 * [`dvc help`](docs/help.md) - Display help for dvc.
-* [`dvc identity`](docs/identity.md) - Update your DevCycle Identity.
+* [`dvc identity`](docs/identity.md) - Print your DevCycle Identity.
```

Comment:
Maybe it should be "View or update" since `dvc identity update` is also an extension of the `dvc identity` command? Looked at the `dvc features` command for reference:)